Cargo.lock

```diff
@@ -1078,6 +1078,7 @@ version = "0.26.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "afc22eff61b133b115c6e8c74e818c628d6d5e7a502afea6f64dee076dd94326"
 dependencies = [
+ "cc",
```

Comment:
I'm pretty sure this was added as a result of me upgrading my rust toolchain this morning.